src/Compilers/CSharp/Test/Semantic/Semantics/RecordTests.cs

```diff
@@ -13199,6 +13941,9 @@ public void IEquatableT_14()
 }";
             var comp = CreateCompilation(source);
             comp.VerifyDiagnostics(
+                // (3,17): error CS8872: 'A.Equals(A)' disallows overriding and containing 'record' is not sealed.
```

Comment:
Consider using diagnostic language like: `'A.Equals(A)' must allow overriding because the containing record is not sealed.`